rero_ils/modules/cli.py

```diff
@@ -225,3 +233,181 @@ def create(infile, pid_type, schema, verbose, dbcommit, reindex):
 
 
 fixtures.add_command(create)
+
+
+@utils.command('check_license')
+@click.argument('configfile', type=click.File('r'), default=sys.stdin)
+@click.option('-v', '--verbose', 'verbose', is_flag=True, default=False)
+@click.option('-p', '--progress', 'progress', is_flag=True, default=False)
+def check_license(configfile, verbose, progress):
+    """Check licenses."""
+    click.secho('Testing licenses in files.', fg='green')
+
+    def get_files_recursive(paths, extensions):
```

Comment:
a lot of common code with the next function. Can you merge it?

Reply:
ok